packages/vscode-extension/.vscodeignore

```diff
@@ -1,22 +1,22 @@
-.vscode/**
-.vscode-test/**
-src/**
+!out/templates/**
+.coverconfig.json
+.github
 .gitignore
+.prettierignore
+.prettierrc.json
+.vscode-test/**
+.vscode/**
 .yarnrc
-**/tsconfig.json
 **/.eslintrc.json
 **/*.map
 **/*.ts
-test-folder/**
-test/**
+**/tsconfig.json
 build/**
-.prettierrc.json
-.prettierignore
-webpack.config.js
-.github
-node_modules
-!out/templates/**
 coverage
-.coverconfig.json
 img/**
```

Comment:
Resources inside img folder will not be included in the package, moving the sample resources to this folder will lost them, right?

Reply:
Images used in [controls/SampleGallery.tsx](https://github.com/OfficeDev/TeamsFx/pull/4908/files#diff-10480cac513d5bc999a7a187df546398f616ce8b9381d7da18fafd200960faff) are actually bundled into client.js by Webpack. So, they are not necessary to be kept in media folder.

Reply:
Will images used in tree view bundled into extension.js by Webpack? If so, does that mean we can also exclude media folder from .vscodeignore?

Reply:
No, they are 2 concepts. Sample gallery is using image like `import xx from "xxx.png"` and renders in a web view. Media folders are used in runtime and since we are not using webview as treeview, the icons in treeview will be rendered dynamically by VS Code.